README.asciidoc

```diff
@@ -31,7 +31,8 @@ upload the exact same artifact (a zip file) into Fusion, so Fusion can host it f
 
 |====================================================
 | Fusion release | SDK version
-| 5.4.0 | link:https://github.com/lucidworks/connectors-sdk-resources/tree/v4.0.0[4.0.0]
+| 5.4.4 | link:https://github.com/lucidworks/connectors-sdk-resources/tree/v4.0.0[4.1.0]
```

Comment:
change `v4.0.0[4.1.0]` to `v4.1.0[4.1.0]`

Reply:
yeah, thanks